app/jobs/fax_application_job.rb

```diff
@@ -1,11 +1,11 @@
 class FaxApplicationJob < ApplicationJob
   def perform(export:)
-    export.execute do |snap_application, logger|
-      receiving_office = snap_application.receiving_office
+    export.execute do |benefit_application, logger|
```

Comment:
to be consistent with your other changes, should this be `snap_application` as well @hartsick ?